onnx/defs/schema.h

```diff
@@ -1059,6 +1059,7 @@ class OpSchemaRegistry final : public ISchemaRegistry {
  public:
   static const std::vector<OpSchema> get_all_schemas_with_history() {
     std::vector<OpSchema> r;
+    r.reserve(4);
```

Comment:
why "4"?

Reply:
I will remove that reserve. The logic was that 4 is better than 0 (because that incurs 3 reallocs to get to capacity=4), but not high enough to waste memory.

Reply:
The logic was that 4 is better than 0 (because that incurs 3 reallocs to get to capacity=4), but not high enough to waste memory. But I will remove that diff.